docs/src/pages/Conformance.md

```diff
@@ -172,7 +172,10 @@ FHIR search modifiers are described at https://www.hl7.org/fhir/R4/search.html#m
 
 Due to performance implications, the `:exact` modifier should be used for String search parameters when possible.
 
-The `:above`, and `:below` modifiers for Token search parameters are not yet supported by the IBM FHIR server. Use of these modifiers will result in an HTTP 400 error with an OperationOutcome that describes the failure.
+The `:above` and `:below` modifiers for Token search parameters are supported, with the following restrictions:
+* The search parameter value must have the form of `[system]|[code]`.
+* The referenced code system must exist in the FHIR registry.
+* The referenced code must exist in the code system.
```

Comment:
this particular one doesn't seem like much of a restriction to me :-)

Reply:
Good point, not really a restriction - removed.